cmd/pp-gen/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/lyraproj/lyra/cmd/goplugin-tf-aws/tfaws"
```

Comment:
Given that this is specific to AWS generation, should we keep this in `cmd/goplugin-tf-aws`?

Reply:
Picking up on the comment below, I reckon we parameterize this. Then the pp-gen tool just takes a `Service`.